workshops/book-dash/book-dash-nov20-report.md

```diff
@@ -0,0 +1,232 @@
+
+### The book dash participants and event plan
+
+Selected from a pool of applications, 20 candidates including two mentors were invited to attend this in-person event (see their names in the group picture below).
+
+![](figures/Nov20-BookDash.png)
+_Zoom screenshot of our community share-out event on 13 November 2020 at the Book Dash. Our Book Dash participants included Laura Acion (Argentina), Emma Karoune (UK), Marilina Santero (Spain/Argentina), Irene Komal (India), Asha Titus (UK), Ismael Kherroubi Garcia (Spain/UK), Martia Mangiarulo (UK/Italy), Laura Carter (UK), Neha Moopen (The Netherlands), Paul Owoicho (UK/Nigeria), Batool Almarzouq (Saudi Arabia/UK), Samuel Guay (Canada), Martina Vila (Germany), Sarah Gibson (UK), Sophia Batchelor (UK), Carlos Martinez-Oritz (The Netherlands), Lupe Canaviri Maydana (Bolivia) and Sangram Keshari Sahu (India). Several community members joined the share-out call on the last day._
+
+A participation support fund was made available for the participants to purchase small equipment or other resources (such as headphone, webcam, second screen, or software) that can improve the quality of their participation and enhance their overall experience of working with us for the entire week.
+They were asked to select at least one development session from the following slots each day (time given in UTC):
+- Development session 1: 08:00 - 10:30
+- Development session 2: 11:00 - 13:30
+- Development session 3: 17:30 - 20:00
+
+A longer break from 13:30 - 17:30 was given to allow people to participate in informal events, self-organised activities, accomplish their day-to-day task outside the Book Dash or take a break as required.
+
+## Pre-dash drop-in calls: 4 November 2020
+
+We hosted 2 pre-dash drop-in calls on 4 November 2020 to ensure that all our participants were familiar with the project, book dash and how they can get the most out of their participation ([HackMD notes](https://hackmd.io/@turingway/bookdash-nov2020-drop-in)).
+The main aims of this call were to get to know our attendees, connect them with others and address their questions before the book dash.
+
+Using a [SMART goal template](https://freespiritpublishingblog.com/wp-content/uploads/2015/08/setting-smart-goals.pdf) (by Danielle R. Schultz [2015]), we also facilitated goal-setting exercise for all the attendees to get them prepared for their participation during the Book Dash week.
+
+Several participants shared their contribution plans and topics they would like to add in _The Turing Way_ that is either missing in the existing chapters or need to be updated. 
+You can read notes from these calls [here](https://hackmd.io/@turingway/bookdash-nov2020-drop-in).
+
+## Book Dash Week: 9-13 November 2020
+
+Each day we will host 3 development sessions and one open slot where participants could plan their time to interact with others, continue working or take a break as per their need.
```

Comment:
"each day we will host" -> "each day we hosted" 

Also this line seems a repeat of the event plan above - consider deleting?